docs/src/components/mdx/headings.tsx

```diff
@@ -40,7 +40,7 @@ export const H3: React.FC<HTMLAttributes<HTMLHeadingElement>> = props => {
   const className = cx(styles.h3, props.className)
 
   return (
-    <Text.Heading size={Text.Heading.sizes.medium} className={className}>
+    <Text.Heading size={Text.Heading.sizes.xsmall} className={className}>
```

Comment:
This change should be reverted looks like xSmall casing didn't change on Heading.Sizes

Reply:
Fixed